contracts/script/utils/ChainIds.sol

```diff
@@ -0,0 +1,11 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.23;
+
+library ChainIds {
+    uint256 public constant MAINNET = 1111; // TBD
+    uint256 public constant ILIAD = 1513;
+    uint256 public constant MININET = 1512;
+    uint256 public constant ODYSSEY_DEVNET = 1315;
+    uint256 public constant ODYSSEY_TESTNET = 1516;
+    uint256 public constant LOCAL = 31337;
```

Comment:
I think local is 1511? @edisonz0718

Reply:
You are right. For local, chain ID of 1511 is used for now.

Reply:
Maybe there is a different understanding of local, that's foundry's `anvil` local network, for tests

Reply:
I added separated variables for LOCAL and FOUNDRY